docs/09_firebase_deployment.md

```diff
@@ -93,13 +93,25 @@ Finally, your Firebase project configured and it's time to configure the Firebas
 
 ## Firebase SDK configuration
 
-To configure the Flutter for Web application to use recently created Firestore Database follow the next steps:
+To configure the Flutter for Web application to use recently created Firestore Database you have two options:
 
+ - use the `auto-generated` Firebase configuration:
```

Comment:
It would be great to mention that this option is used by default, and the user can skip this configuration step if he wants to use this method.